File: commands/pet.js
import { MessageFlags } from '@discordjs/core'
import { SlashCommandBuilder } from '@discordjs/builders'

const data = new SlashCommandBuilder()
        .setName("pet")
        .setDescription("Pet the Banshee Bot")
const execute = async function (interaction, api) {
        api.interactions.reply(interaction.id, interaction.token, { content: "What am I to you, a cow to pet?\nhttps://tenor.com/view/chika-fujiwara-hit-cute-kawaii-anime-gif-13583613", flags: MessageFlags.Ephemeral });
    }

export { data, execute }

File: src/commands/pet.rs
use poise::CreateReply;

use crate::types;

/// Pet the bot... Or not.
#[poise::command(slash_command)]
pub async fn pet(ctx: types::Context<'_>) -> Result<(), types::Error> {
    ctx.defer_ephemeral().await?;
    ctx.send(
        CreateReply::default()
            .content("What am I, some sort of animal for you to pet?\nhttps://media1.tenor.com/m/xc19_U9dSNMAAAAd/chika-fujiwara-hit.gif")
            .ephemeral(true),
    )
    .await?;
    Ok(())
}
